src/states.jl

```diff
@@ -217,6 +217,8 @@ struct PhaseEquil{FT, PS} <: AbstractPhaseEquil{FT}
     param_set::PS
     "density of air (potentially moist)"
     ρ::FT
+    "reference air pressure"
+    p::FT
```

Comment:
By reference what do you mean, exactly?

Reply:
its probably a rebasing error we decided to remove any indication of anelastic or reference